packages/app/src/test-utils.tsx

```diff
@@ -11,12 +11,12 @@ import { expect, type MockInstance, vi } from 'vitest';
 import App from './App';
 import MockProvider from './providers/mock/MockProvider';
 import { type Vis } from './vis-packs/core/visualizations';
-import { type NexusVis } from './vis-packs/nexus/visualizations';
+import { type NxDataVis } from './vis-packs/nexus/visualizations';
 
 interface RenderAppResult extends RenderResult {
   user: ReturnType<typeof userEvent.setup>;
   selectExplorerNode: (name: string) => Promise<void>;
-  selectVisTab: (name: Vis | NexusVis) => Promise<void>;
+  selectVisTab: (name: Vis | NxDataVis) => Promise<void>;
```

Comment:
Is there not typing problems now that `NxNote` is not part of `NxDataVis` anymore?